lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -69,7 +70,7 @@ export class DestCopy implements IBroccoliPlugin {
 				});
 			}
 		});
-		if (!_.isEmpty(this.dependencies)) {
+		if (!_.isEmpty(this.dependencies) && !this.$config.debugLivesync) {
```

Comment:
All I can give is an opinion, as a person, who is not familiar with CLI. I am relying on you to give the best ideas and solutions based on the discussions we have and comments we provide.